src/api/settlements/settlements.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+grand_parent: API Reference
+parent: Settlements
+title: Settlements
+nav_order: 1
+permalink: /api/settlements
+redirect_from:
+  - /settlements
+---
+
+# Settlements
+{:.no_toc}
+
+A settlement is created from completed [Payment Requests][] over a specified period for each supported [Asset Type][] for each [Merchant][].
```

Comment:
Can you also add here "Settlements can only be created if the merchant has a settlement config (and link to settlement config)?